### DISCUSSION TITLE
Github Student Developer Pack Renewal Application Rejected

### DISCUSSION THREAD

**Discussion by Yawarzy** [community/community#61596] [GitHub Education]
### Hi everyone,

Hi everyone, 
I am currently pursuing my Integrated Master's in Computer Applications which is a 5 year Programme, currently I am in the third year. I had applied for the Students Developer Pack last year, our university does not provide us with an edu email, so I had used my personal email uploaded the picture of my School ID, and thankfully my application was accepted. A few weeks ago, I got the email that I need to re-verify my student status, I applied with the same email but this time I was asked to submit the edu email when I selected my university, there was simply no way to continue forward. So instead of selecting my university, I selected my college. And uploaded the College ID. But I instantly got the email that my application was rejected. 
I would be thankful to everyone who can guide me on how should move forward with this.
Thanks.

**Comment by Zweistein2** [comment]
Same here. Second Re-Verification, now asking for an edu email. When I try to register my university manually (so i can use my private email) I have to upload a picture of my university ID... which is then rejected instantly with the following message:

"The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.

Suggested proof to expedite your approval:

    Attendees of this school must verify and use their academic email address."

So here I am with a dated university ID, letter of enrollment, and other documents to prove I'm still a student, but Github just ignores them and wants the edu email I don't have.

**Reply by Yawarzy** [reply]
That's really unfortunate. I have asked my college administration to provide us with the edu emails, once I get the email, will apply again.
